src/assets/styles.scss

```diff
@@ -29,6 +29,8 @@
 @import 'components/_table.scss';
 @import 'components/_button.scss';
 
+@import '/components/_loggerTab.scss';
```

Comment:
This can't have a leading slash, the newer version of sass will not find the file with a leading slash.